README.md

```diff
@@ -42,8 +42,8 @@ fn add(x: u32, y: u32) -> u64 {
 
 #[test]
 fn test_add() {
-    rdescribe("add", |ctx| {
-        ctx.describe("0 <= x + y <= u32::MAX", |ctx| {
+    rdescribe("add", (), |ctx, _| {
```

Comment:
Oh, this is a remainder of an intermediate state of the PR. I need to proof-read and update README.